load-tests/online_shop/deployment/deployment-patch.yaml

```diff
@@ -7,6 +7,7 @@ spec:
     metadata:
       labels:
         logs: "true"
+        load: "true"
```

Comment:
Is the indentation intentional?

Reply:
Yes it is. 

https://github.com/ballerina-platform/ballerina-performance-cloud/pull/3097

Reply:
Ack